ArduSub/inertia.cpp

```diff
@@ -11,7 +11,9 @@ void Sub::read_inertia()
     current_loc.lat = inertial_nav.get_latitude();
 
     // exit immediately if we do not have an altitude estimate
-    if (!inertial_nav.get_filter_status().flags.vert_pos) {
+    nav_filter_status filt_status;
```

Comment:
These sub changes should be separated out into a separate commit.  This applies to various other commits as well so I won't keep repeating this comment.